### ISSUE TITLE
[authoring] - bullet lists with new lines breaks subsequent bullet spacing

### ISSUE THREAD

**Issue by ZaymonFC** [dxos/dxos#5860]
https://github.com/dxos/dxos/assets/12402727/df010732-c4bd-488a-9226-cd007d171394

**Comment by wittjosiah** [comment]
I agree it's a little weird at first, but it's based on the markdown spec and is as designed at the moment. See conversation on the issue I opened here https://github.com/dxos/dxos/issues/5630

**Comment by ZaymonFC** [comment]
Ah I see. One thing that's kinda weird, is that if you make a new tight list lower down on the page, it still adds line breaks for each subsequent bullet.

**Comment by wittjosiah** [comment]
Yeah, I think that's what was throwing me off as well. I guess the question is whether there's a way to differentiate between being inside a tight list or creating a new list. Probably? Will leave this open so we can evaluate that.

**Comment by ZaymonFC** [comment]
My thinking is that if there's any non line-break element in between list elements then it's conceptually a new instance of the bullet list?
